src/components/Table.vue

```diff
@@ -1582,7 +1582,7 @@ export default {
 
     initializeSort() {
       const { enabled, initialSortBy, multipleColumns } = this.sortOptions;
-      const initSortBy = JSON.parse(JSON.stringify(initialSortBy));
+      const initSortBy = JSON.parse(JSON.stringify(initialSortBy || {}));
```

Comment:
If `initialSortBy` is `undefined` default to an empty object to prevent `JSON.parse(undefined)` error from throwing.